File: src/kotlin/luhn/LuhnTest.kt
package luhn

import org.junit.Assert
import org.junit.Test

class LuhnTest {
    @Test
    fun testValidNumber() {
        Assert.assertTrue(check(4111111111111111))
    }

    @Test
    fun testInvalidNumber() {
        Assert.assertFalse(check(4111111111111112))
    }
}

File: src/python/luhn/luhn_test.py
import unittest

from src.python import luhn

class LuhnTests(unittest.TestCase):
    def test_valid_number(self):
        self.assertTrue(luhn.check(4111111111111111))

    def test_invalid_number(self):
        self.assertFalse(luhn.check(4111111111111112))
